src/quiver.mjs

```diff
@@ -373,6 +373,9 @@ QuiverImportExport.tikz_cd = new class extends QuiverImportExport {
             if (settings.get("export.centre_diagram")) {
```

Comment:
This option should be ignored if the standalone option is chosen, because it causes a LaTeX error.